CHANGELOG.md

```diff
@@ -5,6 +5,36 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [2.0.0] - 2021-02-08
+
+Placemat version2 incorporates new features and improvements based on the knowledge we have accumulated over the years.
```

Comment:
I think it is more user-friendly to have a summary of incompatible changes.
How about adding `UPGRADE_v2.md` and adding a link to it here?